CMakeLists.txt

```
@@ -38,7 +40,7 @@ else()
 endif()
 
 find_package(libcosim REQUIRED)
```

Here is the change I alluded to in my earlier comment:
```suggestion
find_package(libcosim REQUIRED CONFIG)
```
It should prevent `find_package()` from picking up the `Findlibcosim.cmake` script generated by Conan.

---

Then FMILIB can't be found. The safest approach here would be to patch proxymfu by providing an explicit FindThrift.

---

https://github.com/open-simulation-platform/proxy-fmu/pull/83